aiida/backends/general/abstractqueries.py

```diff
@@ -45,98 +45,6 @@ def get_duplicate_node_uuids(self):
             """
         return self.raw(query)
 
-    # This is an example of a query that could be overriden by a better implementation,
-    # for performance reasons:
-    def query_jobcalculations_by_computer_user_state(
```

Comment:
Are these just renamed or completely removed? These were here because they could be made faster by custom implementations of the query (how does now `verdi calculation list` get the list of non-finished calculations, and is this fast?

Reply:
I removed them because they were not being used, and they are outdated. `verdi calculation list` uses the `JobCalculation._list_calculations` classmethod, which is also not ideal, but that has been the situation for a long time. Currently, it is the process state that determines "activeness" of a process and hence calculation, the old `calc_state` is merely an additional attribute.